app/views/families/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<a href="families/new">Add a Family</a>
+<p>List of Families:</p>
+<%= render @families %>
+<br>
```

Comment:
Is the `<br>` needed here? or could you use css to give it more space?

Reply:
I added it just for the moment because it looks nicer, but I will fix it properly once strating with css. Thanks!